var/spack/repos/builtin/packages/parallel-netcdf/package.py

```diff
@@ -11,11 +11,25 @@ class ParallelNetcdf(Package):
     version('1.7.0', '267eab7b6f9dc78c4d0e6def2def3aea4bc7c9f0')
     version('1.6.1', '62a094eb952f9d1e15f07d56e535052604f1ac34')
 
+    variant('cxx', default=True, description='Build the C++ Interface')
+    variant('fortran', default=True, description='Build the Fortran Interface')
+    variant('fpic', default=True, description='Produce position-independent code (for use with shared libraries)')
+
     depends_on("m4")
     depends_on("mpi")
 
+    # See: https://trac.mcs.anl.gov/projects/parallel-netcdf/browser/trunk/INSTALL
     def install(self, spec, prefix):
-        configure("--prefix=%s" % prefix,
-                  "--with-mpi=%s" % spec['mpi'].prefix)
+        args = list()
+        if '+fpic' in spec:
+            args.extend(['CFLAGS=-fPIC', 'CXXFLAGS=-fPIC', 'FFLAGS=-fPIC'])
+        if '~cxx' in spec:
+            args.append('--disable-cxx')
+        if '~fortran' in spec:
+            args.append('--disable-fortran')
```

Comment:
No pnetcdf is entirely separate from netcdf.  I consulted pnetcdf build
instructions when making this package
On Apr 22, 2016 3:45 PM, "Ben Boeckel" <notifications@github.com> wrote:

> In var/spack/repos/builtin/packages/parallel-netcdf/package.py:
>
> >      def install(self, spec, prefix):
> > -        configure("--prefix=%s" % prefix,
> > -                  "--with-mpi=%s" % spec['mpi'].prefix)
> > +        args = list()
> > +        if '+fpic' in spec:
> > +            args.extend(['CFLAGS=-fPIC', 'CXXFLAGS=-fPIC', 'FFLAGS=-fPIC'])
> > +        if '~cxx' in spec:
> > +            args.append('--disable-cxx')
> > +        if '~fortran' in spec:
> > +            args.append('--disable-fortran')
>
> Does this need to depends_on netcdf-cxx and netcdf-fortran conditionally
> now?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/LLNL/spack/pull/788/files/e03e87b79186c21c3db084056363bea4db8dba04#r60793132>
>